src/ast/ast_validations.c

```
@@ -1,2478 +1,116 @@
+				return 1;
+			}
+		}
+	}
+	
+	return 0;
+}
+
+// Validate that aliases don't have conflicting labels
+static int _ValidateNodeAliasLabels(AST_Node *pattern, dict *node_labels) {
+	if (!pattern) return 0;
+	
```

Cppcheck (2.21.0)

[error] Memory leak

(memleak)